docs/examples/DataSet/Load old data.ipynb

```diff
@@ -17,8 +17,8 @@
     "import qcodes as qc\n",
     "from qcodes.dataset.legacy_import import import_dat_file\n",
     "\n",
-    "from qcodes.dataset.database import initialise_database\n",
-    "from qcodes.dataset.experiment_container import new_experiment\n",
+    "from qcodes import initialise_database\n",
+    "from qcodes.dataset.experiment_container import load_or_create_experiment\n",
```

Comment:
```suggestion
    "from qcodes import load_or_create_experiment\n",
```

Reply:
I guess we should stick to that but we really should get rid of that importing random mixture of everything from the toplevel module.